Services/CollmexManager.php

```diff
@@ -167,6 +167,21 @@ public function getProducts($prodGroup = null, $priceGroup = null, $onlyModified
         return $response->getObjects();
     }
 
+    public function getStockInfo($onlyModified = 0, $companyId = 1, $systemName = 'Shop')
```

Comment:
We should make this changeable by the developer user in the parameters.yml file. Please use in the meantime 'CoffeeBikeCollmexBundle' as systemName.